resources/lang/pl/store.php

```diff
@@ -24,7 +24,16 @@
     ],
 
     'checkout' => [
+        'cart_problems' => 'Ah, wystąpił problem z twoim koszykiem!',
+        'cart_problems_edit' => 'Naciśnij tutaj, aby go zedytować',
+        'declined' => 'Płatność została anulowana.',
+        'error' => 'Wystąpił problem z zatwierdzaniem twojego zamówienia :(',
```

Comment:
z potwierdzeniem? dokończeniem?

Reply:
na pewno nie zatwierdzaniem bo to jest stan niedokonany a zamówienie w tym momencie już jest rip ;_;

Reply:
Do galkana reklamacje :smirk: